policy/src/main/java/org/apache/brooklyn/policy/ha/ServiceRestarter.java

```diff
@@ -133,6 +135,10 @@ protected synchronized void onDetectedFailure(SensorEvent<Object> event) {
             LOG.warn("ServiceRestarter suspended, so not acting on failure detected at "+entity+" ("+event.getValue()+")");
             return;
         }
+        if (isEntityStopping()) {
+            highlightViolation("Failure detected but entity stopping");
```

Comment:
Should we highlight the violation here? I guess the question is more: is it really a violation when we are in a stopping state?